src/coreclr/interpreter/compiler.cpp

```diff
@@ -3134,6 +3142,25 @@ int InterpCompiler::GenerateCode(CORINFO_METHOD_INFO* methodInfo)
                         m_ip += 5;
                         break;
                     }
+                    case CEE_LOCALLOC:
+                        CHECK_STACK(1);
+#if SIZEOF_VOID_P == 8
+                        if (m_pStackPointer[-1].type == StackTypeI8)
+                            EmitConv(m_pStackPointer - 1, NULL, StackTypeI4, INTOP_MOV_8);
+#endif
+                        AddIns(INTOP_LOCALLOC);
```

Comment:
Does the allocated memory need to be zero-initialized when the method has InitLocals bit set?

Reply:
This is done on mono here https://github.com/dotnet/runtime/blob/main/src/mono/mono/mini/interp/interp.c#L7781.

@kotlarmilos Why was this code simply removed from the code port from mono to coreclr, without any TODO/FIXME ?

Reply:
I was aiming to first get feedback on the `localloc` mechanism before introducing `initLocals`, as I wasn’t sure which flag to check. Sorry about that -- I’ll ensure I explicitly add TODOs in my initial changes.